xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountCurrenciesRequestParams.java

```diff
@@ -0,0 +1,63 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.xrpl.xrpl4j.model.client.XrplRequestParams;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+import org.xrpl.xrpl4j.model.transactions.Address;
+import org.xrpl.xrpl4j.model.transactions.Hash256;
+
+import java.util.Optional;
+
+/**
+ * Request parameters for the account_currencies rippled method.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableAccountCurrenciesRequestParams.class)
+@JsonDeserialize(as = ImmutableAccountCurrenciesRequestParams.class)
+public interface AccountCurrenciesRequestParams extends XrplRequestParams {
+
+    static ImmutableAccountCurrenciesRequestParams.Builder builder() {
```

Comment:
Minor nit (so minor I hate to even bring it up) but we're trying to make the project to use two-spaces for all indentations so we can keep commit noise lower in the general case (looks like you're using 4 spaces). [Checkstyle](https://github.com/XRPLF/xrpl4j/blob/main/checkstyle.xml#L162) will catch this during the build, but the build plugin is only set to `WARN` because a good chunk of the code hasn't been brought into line.

(I'm literally being a meme from a Silicon Valley episode with this comment...bucket-list checked).

Anyway, mind fixing? Otherwise, the PR LGTM!

Reply:
no - i totally understand! i didn't know how to import the .idea project settings into intellij and assumed that that happened automatically. while i was here, I reformatted the `XrplClient` as well, since I touched that file, which accounts for the large number of line changes in the most recent commmit